frontend/src/components/c-summary-charts.vue

```diff
@@ -144,7 +146,7 @@
           span.tooltip-text.right-aligned {{ getPercentileExplanation(j) }}
 
       .summary-chart__ramp(
-        v-on:click="openTabZoomSubrange(user, $event, isGroupMerged(getGroupName(repo)))"
+        v-on:click="openTabZoomSubrange(JSON.parse(JSON.stringify(user))\, $event, isGroupMerged(getGroupName(repo)))"
```

Comment:
I think this line contains a redundant back slash

Reply:
Apologies for missing this, have rectified the issue